awscli/clidocs.py

```diff
@@ -178,6 +188,17 @@ def doc_option(self, arg_name, help_command, **kwargs):
             self._document_nested_structure(argument.argument_model, doc)
         doc.style.dedent()
         doc.style.new_paragraph()
+    
+    def doc_global_option(self, help_command, **kwargs):
+        doc = help_command.doc
+        doc.style.h2('Global Options')
+        global_path = os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            'examples/global/global_options.rst'
+        )
+        with open(global_path, 'r') as global_doc:
+            for line in global_doc:
+                doc.writeln(line)
```

Comment:
1. I tried this but it ends up writing something like `option1option2option3`. I'm not sure where that extra new line is coming from, my current workaround is to remove `\n` at a later point. `doc.write(f"{option}\n"` would be functionally the same as `doc.writeln`. I'll play around with it and see what makes sense.
2. I agree this would be a useful method for `ReSTDocument`. Will implement.